src/index.html

```diff
@@ -3,20 +3,78 @@
   <head>
     <meta charset="UTF-8" />
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
     />
     <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>MOYO Header</title>
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
     />
-    <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1735516800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=QZ6lDmSpPbHeqT6gx2d4ZGNPU8LtHi2aEk~52e~6cOVwfkoMC7XOPL4U4W1xi6nsET5fKBG-6anx9moX1kG2ojI9ybSZVp~wvLo1FQNCSLJ2VQaovKKdEdu9QS~nc1y7eQUjzQ3cc~dcc4TiSv8NTaaU~LO3Y0BqW8AdW8udXTDWtPXujxk0DcitGrla8txLiT95QDpl00-sxZAO7HUlXZnNlstj-lG~aihOdrPfcQoouTWmnrb0pHmmtQ-naqsO-~~dW63bzNqj5-~g6m94uZI1J-6PqsE9uYyrA3EKtq2jquuZCFc~o930E55ToOkD3DptflqLKky-zEe-HWvxzg__"
+          alt="Moyo"
```

Comment:
The `alt` attribute for the image tag is currently set to 'Moyo'. Ensure that the `alt` text accurately describes the image content for better accessibility. If 'Moyo' is the brand name or logo, it might be sufficient, but consider adding more descriptive text if necessary.